framework/src/modules/random/endpoint.ts

```diff
@@ -63,10 +63,36 @@ export class RandomEndpoint extends BaseEndpoint {
 		const seed = ctx.params.seed
 			? Buffer.from(ctx.params.seed, 'hex')
 			: cryptography.utils.generateHashOnionSeed();
-		const count = ctx.params.count ?? 1000000;
+		const count = ctx.params.count ?? MAX_HASH_COMPUTATION;
 		const distance = ctx.params.distance ?? 1000;
 
-		const hashes = cryptography.utils.hashOnion(seed, count, distance) as Buffer[];
+		if (distance > count) {
+			throw new Error('Invalid distance. Distance must be less than or equal to count.');
+		}
```

Comment:
Is this needed? Isn't it always required that count must be a multiple of distance? Checking that should also cover this case, right? 🤔